cfn-resources/cloud-backup-snapshot-export-bucket/test/cfn-test-create-inputs.sh

```diff
@@ -49,28 +49,22 @@ atlasAssumedRoleExternalId=$(atlas cloudProviders accessRoles --projectId "${pro
 jq --arg atlasAssumedRoleExternalId "$atlasAssumedRoleExternalId" \
 	--arg atlasAWSAccountArn "$atlasAWSAccountArn" \
 	'.Statement[0].Principal.AWS?|=$atlasAWSAccountArn | .Statement[0].Condition.StringEquals["sts:ExternalId"]?|=$atlasAssumedRoleExternalId' "$(dirname "$0")/role-policy-template.json" >"$(dirname "$0")/add-policy.json"
-echo cat add-policy.json
 
 #------------ Create aws Iam role-------------------
 
 awsRoleID=$(aws iam get-role --role-name "${roleName}" | jq --arg roleName "${roleName}" -r '.Role | select(.RoleName==$roleName) |.RoleId')
 if [ -z "$awsRoleID" ]; then
-	echo -e "No role found, hence creating the role. Created id: ${awsRoleID}\n"
-else
-	aws iam delete-role-policy --role-name "${roleName}" --policy-name "${policyName}"
-	aws iam delete-role --role-name "${roleName}"
 	awsRoleID=$(aws iam create-role --role-name "${roleName}" --assume-role-policy-document "file://$(dirname "$0")/add-policy.json" | jq --arg roleName "${roleName}" -r '.Role | select(.RoleName==$roleName) |.RoleId')
-	echo -e "FOUND id: ${awsRoleID}\n"
+	aws iam put-role-policy --role-name "${roleName}" --policy-name "${policyName}" --policy-document "file://$(dirname "$0")/policy.json"
+	echo -e "No role found, hence creating the role: ${awsRoleID}\n"
+
+	sleep 30 # Role Arn not returning immediately
 fi
 echo -e "--------------------------------AWS Role creation ends ----------------------------\n"
 
-# Role Arn not returning immediately
-sleep 30
-
 #------------ get Role arn-------------------
 awsArn=$(aws iam get-role --role-name "${roleName}" | jq --arg roleName "${roleName}" -r '.Role | select(.RoleName==$roleName) |.Arn')
 
-aws iam put-role-policy --role-name "${roleName}" --policy-name "${policyName}" --policy-document "file://$(dirname "$0")/policy.json"
```

Comment:
changes make sense to me, curious if you were able to verify these changes as the CI is not running for this resource